util/kustomize/kustomize.go

```diff
@@ -72,7 +72,29 @@ type kustomize struct {
 	noProxy string
 }
 
-var _ Kustomize = &kustomize{}
+var KustomizationNames = []string{"kustomization.yaml", "kustomization.yml", "Kustomization"}
```

Comment:
Any reason to not use RecognizedKustomizationFileNames](https://github.com/kubernetes-sigs/kustomize/blob/447a60903cd142948443a6bd441b2749ad643815/api/konfig/general.go#L10)

Reply:
Would we have to import additional Kustomize Stuff?

Reply:
yea we only have an indirect dependency on `kustomize/api`

Reply:
Going from indirect to direct doesn't worry me too much... I don't feel strongly either way.